Say Their Names/Source/Controller/Base/ServiceClient.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  ServiceClient.swift
+//  Say Their Names
+//
+//  Created by Joseph A. Wardell on 6/2/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+protocol ServiceClient {
```

Comment:
There is `ServiceReferring` protocol already which is exactly the same 😅  you could use that (renaming it if you like `ServiceClient` is a better name)

Reply:
Thanks.  I didn't even see that.  I've switched to using ServiceReferring. This allowed me to get rid of the ServiceClientViewController.